app/src/main/java/openfoodfacts/github/scrachx/openfood/views/LoginActivity.java

```diff
@@ -104,20 +104,18 @@ protected void attemptLogin() {
             return;
         }
 
-        final LoadToast lt = new LoadToast(this);
-        save.setClickable(false);
-        lt.setText(getString(R.string.toast_retrieving));
-        lt.setBackgroundColor(ContextCompat.getColor(this,R.color.blue));
-        lt.setTextColor(ContextCompat.getColor(this,R.color.white));
-        lt.show();
+        Snackbar snackbar = Snackbar
+                .make(linearLayout, "Loading", Snackbar.LENGTH_LONG);
```

Comment:
Can we make the snackbar text come from strings.xml? :)

Reply:
Yeah sure !! I will change it @Karljoones

Reply:
Thanks!